Entities/AHuman.cpp

```diff
@@ -3416,8 +3430,8 @@ void AHuman::Update()
 							minThrowVel = maxThrowVel * 0.2F;
 						}
 						Vector tossVec(minThrowVel + (maxThrowVel - minThrowVel) * GetThrowProgress(), 0.5F * RandomNormalNum());
-						tossVec.RadRotate(m_AimAngle);
-						pMO->SetVel(tossVec.GetXFlipped(m_HFlipped) * m_Rotation);
+						tossVec.RadRotate(m_AimAngle + m_AngularVel * deltaTime);
```

Comment:
I eliminated the rotational factor from thrown devices because it made them impossible to be thrown upwards while prone. Being able to throw grenades while laying down prone would serve as a huge advantage to grenades, considering how little use they have alongside infinite-ammo guns.

The angular velocity is something I added afterwards because I liked how angular momentum was visible in the throw indicator, it also feels more intuitive imo.

Thinking about this gave me ideas for another tweak, but I'll have to test it out first.